GetConnected.html

```diff
@@ -80,34 +100,81 @@
     </div>
   </div>
 </nav>
-  
-<div class="container-fluid text-left" style="padding-left:75px; padding-top: 70px; padding-bottom: 60px;">    
-      <div class="row">
-          <div class="col-xs-6 col-sm-6 col-md-6 col-lg-12">
-              <a href="https://discord.gg/JFq44mJ">
+<script> 
+    function mouseHover(){
+        var color = ["#FFC627"];
+        
+        //divhighlight1
+       document.querySelector(".div-highlight").addEventListener("mouseover", function(){
+           document.querySelector(".div-highlight").style.background = color;})
+
+        document.querySelector(".div-highlight").addEventListener("mouseout", function(){
+           document.querySelector(".div-highlight").style.background = null;})
+
+        //divhighlight2
+       document.querySelector(".div-highlight2").addEventListener("mouseover", function(){
+           document.querySelector(".div-highlight2").style.background = color;})
+
+        document.querySelector(".div-highlight2").addEventListener("mouseout", function(){
+           document.querySelector(".div-highlight2").style.background = null;})
+
+        //divhighlight3
+       document.querySelector(".div-highlight3").addEventListener("mouseover", function(){
+           document.querySelector(".div-highlight3").style.background = color;})
+
+        document.querySelector(".div-highlight3").addEventListener("mouseout", function(){
+           document.querySelector(".div-highlight3").style.background = null;})
+
+        //divhighlight4
+       document.querySelector(".div-highlight4").addEventListener("mouseover", function(){
+           document.querySelector(".div-highlight4").style.background = color;})
+
+        document.querySelector(".div-highlight4").addEventListener("mouseout", function(){
+           document.querySelector(".div-highlight4").style.background = null;})
```

Comment:
I tried a couple things before leaving it as is with the redundant lines above:
I tried using querySelectorAll(), which returns a list of all the matched classes, and then you are supposed to be able to iterate over those matches using a forEach(function()). However, that wasn't working for some reason. 

Leaving the class as just .div-hightlight causes only the first container (discord) to appear with the highlight, because the querySelector method only looks at the first instance of it -- from what I've test at least, I could still be implementing it wrong.

The other attempt I made was just giving an id to the div and then doing documentGetElementById(), but that also gave me issues early on. 

I'll keep playing around with this -- definitely could use improvement, but it does at least work.

Reply:
yeah it should be fine!